sklearn/utils/multiclass.py

```diff
@@ -157,6 +157,24 @@ def is_multilabel(y):
         return len(labels) < 3 and (y.dtype.kind in 'biu' or  # bool, int, uint
                                     _is_integral_float(labels))
 
+def check_non_regression_targets(y):
+    """Determine if target `y` is of non regression type
+
+    For the moment, the following (as defined in type_of_target) are allowed:
+        - 'binary', 'multiclass', 'multiclass-multioutput', 
+        'multilabel-indicator', 'multilabel-sequences'
+        - 'continuous', if only there are only two unique values
```

Comment:
This is not matching the content of the function. I think the content of the function is more correct.